CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## master / unreleased
+
+ - `LastCheckpoint` used to return just the segment name and now it returns the full path.
```

Comment:
Full path might mean absolute path, which may not be true. It returns the full relative path instead of just saying `full path`?

Minor though, no strong feelings.

Reply:
good point! updated